src/components/modals/CustomizeToolbar.tsx

```diff
@@ -209,7 +209,7 @@ const ModalCustomizeToolbar: FC = () => {
       </DropToRemoveFromToolbar>
 
       <p style={{ marginTop: '2em', marginBottom: '2em' }}>
-        &lt;{' '}
+        &lt{' '}
```

Comment:
You fixed the other instance, but missed this one. I encourage you to look over the diff of your PR to make sure everything looks okay. Otherwise we have to go back and forth unnecessarily.

Reply:
Sorry about that, I was under the impression that it was only one,